hypervisor/bsp/include/firmware.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2018 Intel Corporation. All rights reserved.
```

Comment:
I guess these new files should have a copyright year set to 2019?

```suggestion
 * Copyright (C) 2019 Intel Corporation. All rights reserved.
```